src/components/Group/AgreementForm.vue

```diff
@@ -25,6 +25,7 @@
           @blur="$v.edit.content.$touch"
           type="textarea"
           :min-rows="20"
+          @keyup.ctrl.enter="send"
```

Comment:
How does it work? There is no send method defined on the controller, right? I guess it should be the same as the form submit action, "maybeSave".

Reply:
I will check on this as well